plugins/askrene/askrene.c

```diff
@@ -806,22 +773,15 @@ static struct command_result *json_askrene_create_channel(struct command *cmd,
 		return command_param_failed();
 
```

Comment:
I would also add a fool-proof check for this type of query cannot be applied to auto chans:
```suggestion
if (strstarts(layer->name, "auto."))
		return command_fail(cmd, 
		                    JSONRPC2_INVALID_PARAMS,
				    "Command does not apply to auto layers.");
```

Reply:
Also do the same for `json_askrene_disable_channel`, `json_askrene_disable_node`, `json_askrene_age`, `json_askrene_inform_channel`.

Reply:
Agreed! Indeed, this gets done automaticly with the API change.